network-store-server/src/main/java/com/powsybl/network/store/server/QueryCatalog.java

```diff
@@ -95,12 +95,12 @@ public static String buildGetIdentifiablesInContainerQuery(String tableName, Col
         return sql.toString();
     }
 
-    public static String buildDeleteIdentifiableQuery(String tableName) {
-        return "delete from " +
-                tableName +
-                " where " + NETWORK_UUID_COLUMN + " = ?" +
-                " and " + VARIANT_NUM_COLUMN + " = ?" +
-                " and " + ID_COLUMN + " = ?";
+    public static String buildDeleteIdentifiablesQuery(String tableName, int idCount) {
```

Comment:
we usually add this part in other queries:
        if (numberOfValues < 1) {
            throw new IllegalArgumentException(MINIMAL_VALUE_REQUIREMENT_ERROR);
        }
maybe rename idCount => numberOfValues for consistency?